README.md

```diff
@@ -4,7 +4,7 @@
 
 A command-line tool helping the choice of optimal responsive image widths to put in your `srcset` attribute(s).
 
-Install as a global package: `npm install -g cleverage/responsive-image-widths#master`
+Install as a global package: `npm install -g "cleverage/responsive-image-widths#master"`
```

Comment:
```suggestion
Install as a global package: `npm i -g github:cleverage/responsive-image-widths#master`
```
maybe specify it’s github in this case

Reply:
I still get this error if I add `github:` but remove the quotes:

`zsh: no matches found: github:cleverage/responsive-image-widths#master`

It really looks like zsh try to find a `github:cleverage/` folder…

Reply:
6.4.1

Reply:
It looks like it’s related to `zsh` or `oh-my-zsh`:
https://github.com/robbyrussell/oh-my-zsh/issues/31#issuecomment-359728582

Let’s keep it without quotes.